src/resolve/resolved-dns-query.h

```diff
@@ -33,14 +33,6 @@ struct DnsQueryCandidate {
 struct DnsQuery {
```

Comment:
maybe add a comment here somewhere that one is supposed to look at pahole before adding/rearranging the structs blindly